internal/kibana/packages.go

```diff
@@ -27,6 +30,47 @@ func (c *Client) InstallPackage(ctx context.Context, name, version string) ([]pa
 	return processResults("install", statusCode, respBody)
 }
 
+// EnsureZipPackageCanBeInstalled checks whether or not it can be installed a package using the upload API.
+func (c *Client) EnsureZipPackageCanBeInstalled(ctx context.Context) error {
```

Comment:
This is a bit hacky and would be prone to fail if this API would change, maybe add a comment mentioning that is only intended to be used between 8.7.0 and 8.8.2, and is only safe to be used on these versions.

Reply:
Added comment here https://github.com/elastic/elastic-package/pull/2511/commits/4da2bd995f732cf64b12a87999225f74aa9eab5a